java/tools/src/main/java/org/apache/arrow/tools/StreamToFile.java

```diff
@@ -35,6 +35,9 @@
  * Converts an Arrow stream to an Arrow file.
  */
 public class StreamToFile {
+  /**
+   *  Reads a record batchs from in and writes them to out.
```

Comment:
should be "Reads an Arrow stream from in and writes it to out"

Reply:
done.